segmatch/include/segmatch/database.hpp

```diff
@@ -0,0 +1,64 @@
+#ifndef SEGMATCH_DATABASE_HPP_
+#define SEGMATCH_DATABASE_HPP_
+
+#include <string>
+
+#include <segmatch/segmented_cloud.hpp>
+
+namespace segmatch {
```

Comment:
What about a `segmatch::database` namespace? It got confusing at some point with the matches. Might be good to clearly separate the two parts.